composer.json

```diff
@@ -13,14 +13,15 @@
         }
     },
     "require": {
-        "php": ">=5.5",
+        "php": "~5.5|~7.0",
```

Comment:
No big difference between `~` and `^` here, as I do not define the patch version (in that case, I'd rely use `^`, of course)